lib/emscripten/src/lib.rs

```diff
@@ -816,6 +822,12 @@ pub fn generate_emscripten_env(globals: &mut EmscriptenGlobals) -> ImportObject
         "invoke_viid" => func!(crate::emscripten_target::invoke_viid),
         "invoke_viidii" => func!(crate::emscripten_target::invoke_viidii),
         "invoke_viidddddddd" => func!(crate::emscripten_target::invoke_viidddddddd),
+
+        // ucontext.h
+        "_getcontext" => func!(crate::emscripten_target::_getcontext),
```

Comment:
Can we move this functions to `context.rs` (or `ucontext.rs`)?

Reply:
Create a new `ucontext.rs` file? That's ok for me :-)

Reply:
Yup!